src/Microsoft.AspNetCore.Razor.Evolution/CodeGeneration/CSharpLiteralCodeConventions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.Evolution.Legacy;
+
+namespace Microsoft.AspNetCore.Razor.Evolution.CodeGeneration
+{
+    internal class CSharpLiteralCodeConventions : CSharpRenderingConventions
```

Comment:
Most of the files in this directory and in particular all of the *conventions* classes aren't being changed here just move to their own files.